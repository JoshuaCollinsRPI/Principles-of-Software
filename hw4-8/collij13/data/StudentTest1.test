r
Hall Hall
DCC
r
W
Hall Hall
m
q
