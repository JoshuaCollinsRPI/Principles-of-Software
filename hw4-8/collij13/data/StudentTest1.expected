First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from Hall Hall to DCC:
	Walk West to (Intersection 103)
	Walk NorthWest to (Public Safety)
	Walk West to (Intersection 141)
	Walk SouthWest to (CII)
	Walk SouthWest to (DCC)
Total distance: 395.569 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [W]
b lists all buildings
r prints directions for the shortest route between any two buildings
q quits the program
m prints a menu of all commands
